### PR TITLE
Update base homepage

### PR DESCRIPTION
**Pull request by sayo96825**
base page with link states and updated main home page

### CODE REVIEW

File: LIMS_IMAGE/web/templates/home.html [hunk 1]
```diff
@@ -1,14 +1,23 @@
-{% extends 'base.html' %} {% block title %}LIMS Home{% endblock %}
+{% extends 'base.html' %}
+{% load bootstrap5 %}
+{% load static %} 
+{% block title %}LIMS Home{% endblock %}
+<link rel="stylesheet" href="{% static 'css/home.css' %}"/>
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/home.html:5]: This should be inside a head tag, as below
```
<head>
  <link rel="stylesheet" href="{% static 'css/home.css' %}"/>
</head>
```

File: LIMS_IMAGE/web/templates/home.html [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
-  </div>
+  {% block content %}
+    <div class="container p-4">
+      <h1>LIMS Home</h1>
+      {% if user.is_authenticated %} Hi {{ user.first_name }}!
+      <p><a href="{% url 'logout' %}">Log Out</a></p>
+      {% else %}
+      <p><i>You are not logged in</i></p>
+      <section id = "main">
+        <h2>Welcome to the LIMS home page</h2>
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/home.html:15]: we can remove this line

File: LIMS_IMAGE/web/templates/home.html [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+        <div class ="container p-4">
+            <img src ="{% static 'image/images2.jpg' %}" alt ="Background image with Laboraroty theme" height = 500px width = 800px>
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/home.html:17]: this image should have a meaningful name like "homepage_image.jpg"

Comment (shepkeira) [inline LIMS_IMAGE/web/templates/home.html:17]: don't set both height and width, this can lead to image stretching and looking odd. Set one or the other

**Review by shepkeira — CHANGES_REQUESTED** [review]

**Review by sselin-co — CHANGES_REQUESTED** [review]
Looks good to me! The styling should be applied to the customer and employee home page as well, since the styling currently is only applied to the anonymous homepage.

**Comment by shepkeira** [comment]
> Looks good to me! The styling should be applied to the customer and employee home page as well, since the styling currently is only applied to the anonymous homepage.

@sselin-co we have separate tickets for the other home pages, so I think that can wait till those tickets (#259 and #260 )

**Comment by sselin-co** [comment]
> > Looks good to me! The styling should be applied to the customer and employee home page as well, since the styling currently is only applied to the anonymous homepage.
> 
> @sselin-co we have separate tickets for the other home pages, so I think that can wait till those tickets (#259 and #260 )

Thanks for catching that! None issue then 🙂

**Review by sselin-co — APPROVED** [review]
Approved👍

**Review by shepkeira — CHANGES_REQUESTED** [review]
Please remove the other version of the image without the name change, so we don't have 2 of the same image with different names

File: LIMS_IMAGE/web/templates/home.html (hunk 2 above)
Comment (shepkeira) [reply LIMS_IMAGE/web/templates/home.html]: Sorry my comment was unclear the line
```<h2>Welcome to the LIMS home page</h2>```
line not the the
```<section id = "main">```
line

**Review by shepkeira — CHANGES_REQUESTED** [review]

**Comment by sayo96825** [comment]
I tried to remove the old image file and removed the <h2>.
Can you please check again?
Thank you so much 🙏

**Review by shepkeira — APPROVED** [review]

**Pull request merged**